server/config/database.config.js

```diff
@@ -0,0 +1,10 @@
+const dbConfiguration = {
+  dev: {
+    username: "postgres",
+    password: "1002",
+    host: "127.0.0.1",
+    port: "5432",
+    database: "messenger",
+  },
+};
+module.exports = { dbConfiguration };
```

Comment:
A file like this should probably be included in your `.gitignore`, just make sure there are instructions for recreating it in your README.